### PR TITLE
Resizable images

### PR DESCRIPTION
**Pull request by elpoelma**
This PR introduces the resizable images feature. Resizable images are implemented using an ember-node, when selected, handles are shown at the corners and sides of the image in order to resize it.

Specifically, this PR contains the following:
- A reworked `image` node-spec and an `image` node-view.
- The addition of an optional `stopEvent` attribute which can be provided when configuring an ember-node, it can be used to override the default behaviour of the `stopEvent` attribute of the ember-node node-view.

This PR is a solution to https://binnenland.atlassian.net/browse/GN-4142

### CODE REVIEW

**Review by lagartoverde — APPROVED** [review]
Looks good to me 👍

**Review by usrtim — APPROVED** [review]
LGTM

File: addon/components/plugins/image/node.ts [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+        });
+      }
+    }
+  }
+
+  onMouseDown = (event: MouseEvent) => {
+    event.preventDefault();
+    event.stopPropagation();
+    const handleType = Object.keys(this.handleReferences).find(
+      (key: HandleType) => this.handleReferences[key] === event.target
```
[the remark this answers is not in the document]
Comment (elpoelma) [reply addon/components/plugins/image/node.ts:178]: Yeah you're right, I will look if I find another way.

[the remark this answers is not in the document]
Comment (elpoelma) [reply addon/components/plugins/image/node.ts]: Improved it a bit :)

**Review by abeforgit — CHANGES_REQUESTED** [review]
the first time you use the left/right/top/bottom handles, the image resizes proportionally, the second time it doesn't

both ff and chrome

**Pull request merged**